frontends/p4/simplifyDefUse.cpp

```diff
@@ -30,9 +31,10 @@ namespace {
 class HasUses {
     // Set of program points whose left-hand sides are used elsewhere
     // in the program together with their use count
-    std::set<const IR::Node *> used;
+    absl::flat_hash_set<const IR::Node *> used;
 
     class SliceTracker {
+        // FIXME: Can squeeze this bool into lower bit of pointer
```

Comment:
Personally, I wouldn't go there unless we have a very good reason to. I do realize the `bool` makes the struct 8 bits larger, but unless we know that significantly limits us I think the cost of tainting a pointer is too much.

Reply:
Yes, this is why it's just some observation. I do not know the particular impact here.